src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -281,6 +285,69 @@ export default class PlotsTab extends React.Component<IPlotsTabProps,{}> {
         }
         return ret;
     }
+    @computed get quickConfigurations(): JSX.Element {
+        if (!this.dataTypeOptions.isComplete ||
+            !this.dataTypeToDataSourceOptions.isComplete ||
+            !this.horzAxisDataPromise.isComplete ||
+            !this.props.store.samplesByDetailedCancerType.isComplete ||
+            !this.props.store.mutations.isComplete
+        ) {
+            return (
+                <div className="quickPlotsContainer">
```

Comment:
would like to see the wrapper container div in only once place and contents (loader or UI) renderend conditionally inside it.